src/reducer.ts

```
@@ -0,0 +1,53 @@
+    case 'toggleTicketStatus': {
+      return tickets.map((ticket) => (
+        ticket.id === action.id
+          ? { ...ticket, status: ticket.status === 'open' ? 'closed' : 'open' }
+          : ticket
+      ));
+    }
+
+    case 'addComment': {
+      
+      const commentIds = tickets.reduce((commentIds, ticket) => [
+        ...commentIds,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)